plugins/telegram-assets/package.json

```diff
@@ -0,0 +1,19 @@
+{
+  "name": "@anticrm/telegram-assets",
+  "version": "0.6.0",
+  "main": "src/index.ts",
+  "author": "Anticrm Platform Contributors",
+  "license": "EPL-2.0",
+  "scripts": {
+    "build": "",
+    "build:docs": "api-extractor run --local",
+    "lint": "ts-standard src",
+    "lint:fix": "ts-standard --fix src",
+    "format": "prettier --write 'src/**/*.{ts*,js*,yml}' && ts-standard --fix src"
```

Comment:
Let's add clean code for new code. There is no dependency to ts-standard or prettier in plugin.